src/test/java/CRTGearRatioUtilTestingStuff.java

```diff
@@ -0,0 +1,43 @@
+import frc.robot.utils.CRTGearRatioUtil;
+import org.junit.jupiter.api.Test;
+import java.util.Optional;
+
+public class CRTGearRatioUtilTestingStuff {
+  @Test
+  void test() {
+    assert checker(44, 42, 22);
+    assert checker(44, 26, 22);
```

Comment:
Check symmetry, e.g. does it also work for `42, 44, 22`?

Reply:
It doesn't, which is correct behavior.